docs/migrations.rst

```diff
@@ -0,0 +1,55 @@
+..  _migrations:
+
+Migrations
+=========================
+
+Contessa stores quality check results to the database in the defined data model. 
+These data or data model are sometimes changed and when it happens, we need to migrate data and data model to the new version.
+
+When to migrate
+-------------------------
+The schema change is not present in all Contessa version changes. Please, be careful and read the changelog to identify when you need to do the migration.
```

Comment:
can we add to changelog that we have breaking change ? :D i think we forgot that.

Reply:
we'll add it to the changelog when we'll increase version number. Or not?

Reply:
ok np